.github/workflows/testing.yml

```
@@ -161,7 +161,7 @@ jobs:
 
   build-n-publish:
     name: Packaging
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
```

I would actually prefer to pin this to a particular target, rather than a moving one - for builds, we really want as much stability as possible. We can absolutely try something the current `latest` though:
```suggestion
    runs-on: ubuntu-20.04
```